ci/deploy_mz

```diff
@@ -0,0 +1 @@
+deploy-mz
```

Comment:
Using underscores in the pipeline directories was intentional, because Python modules can't have hyphens in them. It annoys me too, but it seemed like the best option. I think there's at least one place where we need to import across directories that forces the underscore naming scheme.